test/liboccamize/main-and-library/README.txt

```diff
@@ -0,0 +1,6 @@
+Automatic library specialization mode:
```

Comment:
This is a bit confusing. Is `lib_spec` the library for which we add a dummy main and specialize against it?
In that case, shouldn't `main` be empty?

Reply:
For this, lib_spec would be a list of library bitcodes which the main program would be linked to. This was just an additional alternative mode where instead of the user specifying entry points, we would first extract all the external functions declearations in the main module. And create nd calls in the library modules (in lib_spec) and make dummy mains in the libraries accordingly. I hope that makes sense.

Reply:
In the case where we want to manually specify entry points to a library module, the main key in the manifest would be the library bitcode, and through slash we would specify entry points with —entry-point=...